src/miner.cpp

```diff
@@ -138,27 +147,6 @@ CBlockTemplate* CreateNewBlock(const CScript& scriptPubKeyIn)
                                 ? nMedianTimePast
                                 : pblock->GetBlockTime();
 
-        UpdateTime(pblock, Params().GetConsensus(), pindexPrev);
```

Comment:
This call to UpdateTime was added in BIP101, and was kept when BIP101 was reverted and BIP109 merged.